hazelcast/src/main/java/com/hazelcast/executor/impl/DistributedExecutorService.java

```diff
@@ -146,6 +151,7 @@ public void destroyDistributedObject(String name) {
         shutdownExecutors.remove(name);
         executionService.shutdownExecutor(name);
         statsMap.remove(name);
+        executorConfigCache.remove(name);
```

Comment:
Remove `ExecutorConfig` from cache, if one exists, when executor proxy is destroyed.